<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exakova TextFlow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1f1f1f;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin" stdset="0">
    <number>0</number>
   </property>
   <item>
    <widget class="QMenuBar" name="menuBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
            QMenuBar {
                background-color: #1f1f1f;
                color: white;
                border: none;
                padding: 2px 0px;
            }
            QMenuBar::item {
                padding: 2px 8px;
            }
            QMenuBar::item:selected {
                background-color: #404244;
            }
            QMenu {
                background-color: #292a2b;
                color: white;
                border: 1px solid #1b1b1b;
            }
            QMenu::item {
                padding: 4px 24px;
            }
            QMenu::item:selected {
                background-color: #235c96;
                border: 1px solid #50a0f0;
            }
          </string>
     </property>
     <widget class="QMenu" name="menuFile">
      <property name="title">
       <string>File</string>
      </property>
      <addaction name="actionOpen"/>
      <addaction name="actionExport"/>
     </widget>
     <addaction name="menuFile"/>
    </widget>
   </item>
   <item>
    <widget class="QTabBar" name="tabBar" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
            QTabBar {
                background: #292a2b;
                border: none;
            }
            QTabBar::tab {
                background: #292a2b;
                color: white;
                min-width: 120px;
                padding: 3px 6px 3px 6px;
                border: 1px solid #1b1b1b;
                margin-right: 2px;
            }
            QTabBar::tab:selected {
                background: #235c96;
                border-bottom: 1px solid #50a0f0;
            }
          </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>4</number>
     </property>
     <widget class="QWidget" name="leftPanel" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>16</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
              background-color: #292a2b;
              border-right: 1px solid #1b1b1b;
            </string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="pdfView">
      <property name="styleSheet">
       <string notr="true">
              background-color: #1f1f1f;
              border: none;
            </string>
      </property>
     </widget>
     <widget class="QScrollBar" name="rightScrollBar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>16</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
              QScrollBar {
                  background: #292a2b;
                  width: 16px;
              }
              QScrollBar::handle {
                  background: #404244;
                  min-height: 20px;
              }
              QScrollBar::handle:hover {
                  background: #505254;
              }
              QScrollBar::add-line, QScrollBar::sub-line {
                  background: none;
              }
            </string>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTabBar</class>
   <extends>QWidget</extends>
   <header>qtabbar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
